Tests/HummingbirdTests/TrieRouterTests.swift

```diff
@@ -29,6 +29,30 @@ class TrieRouterTests: XCTestCase {
         XCTAssertEqual(trie.resolve("/Users/jane/bin")?.value, "test3")
     }
 
+    func testPathParsing() {
+        let trieBuilder = RouterPathTrieBuilder<String>()
+        func getFirstChildElement(_ path: String) -> RouterPath.Element? {
+            trieBuilder.root.children.first(where: { $0.key == path })?.children.first?.key
+        }
+        trieBuilder.addEntry("test1/:param", value: "*")
```

Comment:
Just a quick question: shouldn't we have a test for something like this:

```swift
"foo/{fooId}"
"foo/{fooId}/bar/{barId}
```

Reply:
This test is just there to check the parsing of individual path components. I've added another test in `testGetParameters` which does pretty much what you are asking